### PR TITLE
add dev refactorings

### PR DESCRIPTION
**Pull request by thde**
This PR adds a Dockerfile, docker-compose, sentry and updates the Github client.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM docker-registry.nine.ch/ninech/ruby:trusty
```
Comment (born4new) [inline Dockerfile]: As this is a public project, we might want to use another image here than one we have on our private repo :)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+MAINTAINER development@nine.ch
```
Comment (born4new) [inline Dockerfile]: `MAINTAINER` is deprecated: we have to use labels now

File: Dockerfile (hunk 1 above)
Comment (thde) [reply Dockerfile]: Good point 😄

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FROM ruby:2-alpine
+LABEL maintainer="development@nine.ch"
+
+COPY Gemfile Gemfile.lock VERSION ./
+RUN bundle install --jobs 2
+
+COPY . .
```
Comment (born4new) [inline Dockerfile]: I would either have an explicit `WORKDIR` set, or copy the content to a specific destination (`/app` say), as of now, it's not clear where the files are being copied in the container...

File: config.ru [hunk 4]
```diff
@@ -1,3 +1,14 @@
-require './app'
+require 'raven'
 
-run Sinatra::Application
+Raven.configure do |config|
+    config.dsn = ENV['SENTRY_DSN']
+    #config.environments = %w[production staging]
```
Comment (born4new) [inline config.ru]: Could we remove these comments or uncomment them? Either way, fine by me 😆

**Pull request merged**